RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -132,12 +132,35 @@ class PumpOpsSynchronous {
             throw lastError
         }
     }
+    
+    private func pumpResponding() -> Bool {
+        do {
+            let msg = makePumpMessage(.GetPumpModel)
+            let response = try sendAndListen(msg, retryCount: 1)
+            
+            if response.messageType == .GetPumpModel && (response.messageBody as? GetPumpModelCarelinkMessageBody) != nil {
```

Comment:
`(response.messageBody as? GetPumpModelCarelinkMessageBody) != nil` is equivalent to `response.messageBody is GetPumpModelCarelinkMessageBody`